clients/lsp-clangd.el

```diff
@@ -226,7 +226,9 @@ returned to avoid that the echo area grows uncomfortably."
                                        "\n")
                                       "```")) nil t nil)
           (progn (narrow-to-region (match-beginning 2) (match-end 2))
-                 (lsp--render-element (lsp-clangd-join-region (point-min) (point-max))))
+                 (lsp--render-element
+                  (ht ("language" "cpp")
```

Comment:
You should use `lsp-make-marked-string` instead.

Reply:
Fixed it.